core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -476,6 +573,32 @@ public Sender build() {
             return sender;
         }
 
+        /**
+         * Disables automatic flushing of buffered data.
+         * <p>
+         * Sender buffers data locally before sending it to a server. This method disables automatic flushing and
+         * you must explicitly call {@link #flush()} to send buffered data to a server. It also disables auto-flushing
+         * on close. This gives you a fine control over batching behaviour.
+         * <p>
+         * You must ensure maximum buffer capacity is large enough to accommodate all locally buffered data.
+         * <p>
+         * This method is only used when communicating over HTTP protocol. It's illegal to call this method when
```

Comment:
Out of curiosity.. why it not supported with TCP?

Is the impl for flushing separate and only the new stuff supports it?

Reply:
yes. I felt like investing into the TCP transport wasn't worth it.